server/templates/ac/user_confirm.html

```diff
@@ -11,12 +63,20 @@
 		</script>
 	</head>
 	<body>
-		Mattermost user {{ .MattermostDisplayName }} is requesting to connect as Jira user {{ .JiraDisplayName }}
-		<form id="user_connect" name="user_connect" action="{{ .ConnectSubmitURL }}">
-			<input type="hidden" id="{{ .ArgJiraJWT }}" name="{{ .ArgJiraJWT}}" value="none"/>
-			<input type="hidden" id="{{ .ArgMMToken }}" name="{{ .ArgMMToken}}" value="{{ .MMToken }}"/>
-			<input type="submit" value="Approve"/>
-		</form>
+		<div class="flex-parent">
+				<h3>
+					Mattermost user is trying to connect to Jira.
+				</h3>
+				<div class="accounts-container">
```

Comment:
@jasonblais @asaadmahmood @crspeller 1/5 I was just going through the flow, and we should maybe include some extra copy here that explains the ramifications of clicking Approve from the security point of view (MM user will be able to xxx without re-authenticating to Jira)